src/docfx/build/page/BuildPage.cs

```diff
@@ -166,7 +166,9 @@ private static SystemMetadata CreateSystemMetadata(ErrorBuilder errors, Context
                 = context.DocumentProvider.GetDocumentId(context.RedirectionProvider.GetOriginalFile(file.FilePath));
 
             (systemMetadata.ContentGitUrl, systemMetadata.OriginalContentGitUrl, systemMetadata.OriginalContentGitUrlTemplate)
-                = context.ContributionProvider.GetGitUrl(file.FilePath);
```

Comment:
So there won't be a case that only one of the three is specified, but the others are expected to be generated by DocFX?

Reply:
There's no mix-usage case for this. 
Either all **user-defined**: special SDP case, e.g. powershell 
Or all **system-generated**